acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -6,5 +6,111 @@ def ruby_command(host)
       end
       module_function :ruby_command
     end
-  end
-end
+
+    module CAUtils
+      def initialize_ssl
+        hostname = on(master, 'facter hostname').stdout.strip
+        fqdn = on(master, 'facter fqdn').stdout.strip
+
+        if master.use_service_scripts?
```

Comment:
Hi @joshcooper - I've copied the Puppet pre-suite file here; but I'm unsure what the context of the phrase "use service scripts" is? What does it mean, and what are the cases for using it or not using it? Can I simplify by not having 015_PackageHostsPresets.rb and making the code here unconditional (i.e. always stop puppetserver at this point) or does that cause a problem in some cases?